### DISCUSSION TITLE
https://notes.alexkehayias.com/46-of-developers-code-comes-from-github-copilot/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#9] [Announcements]
# https://notes.alexkehayias.com/46-of-developers-code-comes-from-github-copilot/

A blog post from GitHub says that their code assistance tool, Copilot, is behind 46% of developers' code, up from 27%.

https://notes.alexkehayias.com/46-of-developers-code-comes-from-github-copilot/

**Comment by simonbengtsson** [comment]
Would be very interesting to see my own stats on code produced with copilot and how it changes over time.
